JASP-Desktop/po/jasp_nl.po

```diff
@@ -1644,7 +1644,7 @@ msgstr "Moderator factor 2"
 
 msgctxt "Ancova|"
 msgid "Nonparametrics"
-msgstr "Verdelingsvrije toetsen"
+msgstr "Niet-parametrische toetsen"
```

Comment:
Als we [wikipedia](https://nl.wikipedia.org/wiki/Verdelingsvrije_statistiek) moeten geloven:
> Een verdelingsvrije toets wordt ook wel een parametervrije toets genoemd, maar die naam is verwarrend. Het is namelijk een methode waar best parameters in mogen voorkomen. Een betere vertaling van het Engelse non parametric test zou niet-parametrische toets zijn: het is namelijk niet een parametrische toets. In het Nederlands is de officiële aanduiding beduidend accurater: verdelingsvrije toets. 

Wellicht is deze aanpassing niet geheel nodig? (Ik ken deze termen alleen uit het Engels, dus ik weet niet wat gebruikelijker is.)

Reply:
Ik ken de term ook alleen in het Engels... Verdelingsvrij is inderdaad accurater dan niet-parametrisch — aanpassing is dus niet nodig.